lib/helpers.js

```diff
@@ -60,11 +60,13 @@ module.exports.getFirstWord = function (message) {
 };
 
 module.exports.isYesResponse = function (message) {
-  const trimmed = this.getFirstWord(message.toLowerCase().trim());
+  const trimmedMsg = message.toLowerCase().trim();
   const yesResponses = process.env.GAMBIT_YES_RESPONSES ?
     process.env.GAMBIT_YES_RESPONSES.toLowerCase().split(',') : ['yes'];
 
-  return yesResponses.indexOf(trimmed) >= 0;
+  const matchRegex = new RegExp(`^(${yesResponses.join('|')})$`);
```

Comment:
Ok, just checking it's still the same. It'd be nice to have the readable comment, `// Check if string starts with a Yes response`, but not necessary for merge.